samples/weather_function/readme.md

```diff
@@ -0,0 +1,33 @@
+# Weather Function Sample
+
+This is a Serverless Function that fetches the current weather information from the Open Weather Map [service](https://openweathermap.org/current) via HTTP. Part of the Function's output includes HTTP response headers, and the HTTP response body. This sample is designed to be deployed on OpenFaaS via a Docker image.
+
+
+# Building Docker Image
+
+It will be assumed that Docker is already installed.
+
+1. Clone the Kotlin Native Git repository: `git clone https://github.com/JetBrains/kotlin-native ~/repos/kotlin-native`
+2. Change working directory to the sample: `cd ~/repos/kotlin-native/samples/weather_function`
+3. Create the **openweathermap_key.txt** file and append your [Open Weather Map API key](https://openweathermap.org/appid) to the file
+4. Build the *weather* Docker image by running the shell script: `./build_image`
```

Comment:
Once you convert to the YAML file format for OpenFaaS you can use the CLI do to this build instead (which is preferred)

```
faas-cli build -f weather-example.yml
```